test/integration/test.js

```diff
@@ -1,43 +1,43 @@
-import React from "react"
-import { loadData, SimpleData, SimpleDocument } from "../../dist/index.js"
-import { temporaryDirectory } from 'tempy'
+import { loadData, SimpleData } from "../../dist/index.js"
+import { temporaryDirectoryTask } from 'tempy'
+import * as Plot from "@observablehq/plot"
 
-const simpleData = await loadData("data/employees.csv")
+const simpleData = await loadData({
+    path: "data/employees.csv", 
+    verbose: true, 
+    logParameters: true
+})
 
-simpleData.setDefaultOptions({ logs: true, logOptions: true, logParameters: true })
 
-// console.log(simpleData.data)
-// console.log(simpleData.keys)
-// console.log(simpleData.options)
-simpleData.getArray("Name")
-simpleData.getUniqueValues("Job")
+simpleData.getArray({key: "Name"})
+simpleData.getUniqueValues({key: "Job"})
 
 
 simpleData
-    .describe({ showDataNoOverwrite: true })
+    .describe()
     .formatAllKeys()
-    .renameKey("departementOrUnit", "unit")
-    .renameKey("endOfYearBonus", "bonus")
-    .checkValues({ showDataNoOverwrite: true })
-    .excludeMissingValues("name")
+    .renameKey({oldKey: "departementOrUnit", newKey: "unit"})
+    .renameKey({oldKey: "endOfYearBonus", newKey: "bonus"})
+    .checkValues()
+    .excludeMissingValues({key: "name"})
     .excludeMissingValues()
-    .addKey("firstName", item => item.name.split(",")[1].trim())
-    .removeKey("name")
-    .replaceValues("bonus", "%", "", "partialString")
-    .replaceValues("bonus", ",", ".", "partialString")
-    .valuesToFloat("bonus")
-    .modifyValues("bonus", val => val / 100)
-    .modifyItems("bonus", item => item.salary * item.bonus)
-    .roundValues("bonus", { fractionDigits: 2 })
-    .valuesToInteger("unit")
-    .valuesToString("unit")
-    .valuesToDate("hireDate", "%d-%b-%y")
-    .datesToString("hireDate", "%Y-%m-%d")
-    .filterValues("bonus", val => val >= 100)
-    .filterItems(item => item.hireDate > "2002-01-01" && item.unit !== "20")
-    .sortValues("salary", "descending")
-    .sortValues("bonus", "ascending")
-    .selectKeys(["firstName", "job", "bonus"], { showDataNoOverwrite: true })
+    .addKey({key: "firstName", func: item => item.name.split(",")[1].trim()})
```

Comment:
I know func was the name of the parameter but it wasn't supposed to be public. I wonder what we could use that would be clear for our users. Maybe "values"? "valuesGenerator"?